Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/UH9A-Data-Integrity/data-integrity

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -x
+
+pod() {
+echo $CI_JOB_ID
+###clone e2e-openshift-repo
+echo "cloning e2e-openshift repo*************"
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'git clone -b OpenEBS-base https://github.com/openebs/e2e-openshift.git'
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/UH9A-Data-Integrity/data-integrity node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
+}
+
+node() {
+source ~/.profile
+gittoken=$(echo "$github_token")
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+commit_id=$(echo $3)
+case_id=UH9A
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bash Openshift-EE/utils/e2e-cr jobname:data-integrity-jiva jobphase:Waiting init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+bash Openshift-EE/utils/e2e-cr jobname:data-integrity-jiva jobphase:Running
+
+echo "*******Deploying Busybox-sts Application****"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=fio-data-integrity-jiva metadata="")
+echo $test_name
+cd litmus
+echo "Running the litmus test for Busybox Deployment.."
+sed -i -e 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' apps/fio/tests/data-integrity/run_litmus_test.yml   
```

Comment:
@sushma1118 Instead of doing sed functions directly into the run_litmus_test can you copy the content of the file and perform the sed commads.  This is to ensure that cstor and jiva test has to run in parallel.